CIP-DRep-Credential/README.md

```diff
@@ -0,0 +1,393 @@
+---
+CIP: ???
+Title: On-Chain DRep Credential & Extended Governance NFTs
+Status: Draft
+Category: Tokens
+Authors:
+  - "Derrick Oatway <alpine@malamaproject.org>"
+Implementors:
+  - DRep Collective
+Discussions:
+  - "https://github.com/cardano-foundation/CIPs/pull/1007"
+Created: "2025-03-01"
+License: CC-BY-4.0
+---
+
+## Abstract
+
+> "**Poets are the unacknowledged legislators of the world.**"  
+> — Percy Bysshe Shelley, _A Defence of Poetry_ (1821)
+
+This CIP introduces structured **on-chain** metadata standards to enhance transparency, accountability, and effectiveness in Cardano’s governance. Building on **CIP-1694**, it provides new mechanisms for Delegated Representatives (DReps) and delegators to record, discover, and analyze governance data in a standardized way. Leveraging the **CIP-68** datum-based NFT model, it defines three specialized token types:
+
+1. **DRep Credential** – An on-chain credential NFT referencing both on- and off-chain metadata (per **CIP-119**), with optional fields (e.g., roles, sectors) for DReps to self-report.
```

Comment:
might be worth considering this standard for all governance voters, so also SPOs and CC

Reply:
Sure thing!